mllib/src/main/scala/org/apache/spark/mllib/feature/ChiSqSelector.scala

```diff
@@ -197,3 +197,28 @@ class ChiSqSelector @Since("1.3.0") (
     new ChiSqSelectorModel(indices)
   }
 }
+
+/**
+ * Creates a ChiSquared feature selector by False Positive Rate (FPR) test.
```

Comment:
Is there any link to document what this means? I had to double-check it means what I think it means and could only find indirect references. We might very briefly explain it as a bound on the likelihood that the feature only by chance appears to be predictive, given the data, and even give an indicative common value like 0.05.